.gitignore

```diff
@@ -81,4 +81,4 @@ yarn-error.log
 .idea/
 
 #CNAME
-static/CNAME
+static/CNAME
```

Comment:
there is an error for EOL
please add a new line at the end of this

Reply:
thanks, sure will do.